ros/src/tl_detector/tl_detector.py

```diff
@@ -90,7 +90,7 @@ def image_cb(self, msg):
             self.state = state
         elif self.state_count >= STATE_COUNT_THRESHOLD:
             self.last_state = self.state
-            light_wp = light_wp if state == TrafficLight.RED else -1
+            light_wp = light_wp if state == TrafficLight.RED or state == TrafficLight.YELLOW else -1
```

Comment:
Change looks good.

The code compiles and runs on my end, although I got a few strange warnings. Please verify that this code builds without any errors or warnings for you (I get no warnings in my personal repo, so not sure what's different here).